demos/sslecho/makefile

```diff
@@ -0,0 +1,11 @@
+PROG ?= sslecho
+
+all: $(PROG)
+
+# Debug version.
+#
+$(PROG): main.c
+	$(CC) -O0 -g3 -W -Wall -I/usr/include/openssl -L/usr/lib/x86_64-linux-gnu -o $(PROG) main.c -lssl -lcrypto
```

Comment:
Might be nice to give the ability to specify alternative locations, e.g. see the Makefile in the bio demo